### ISSUE TITLE
Not all pages get backlinks

### ISSUE THREAD

**Issue by LBeukennoot** [six-two/mkdocs-backlinks-section-plugin#3]
In my mkdocs project im using this plugin (works great btw!), only problem is, for some reason backlinks don't show up on some pages.

This page ([Quests](https://lbeukennoot.github.io/dnd-icewind-dale/Quests/Quests/)) gets referred to 18 times, but it doesn't show any backlinks. For example, [this](https://lbeukennoot.github.io/dnd-icewind-dale/Quests/Archin%20Brotherhood%20is%20slecht%21/) page refers to [Quests](https://lbeukennoot.github.io/dnd-icewind-dale/Quests/Quests/).

Pages with a smaller amount of references ([Bremen](https://lbeukennoot.github.io/dnd-icewind-dale/Villages/Bremen%20%E2%9D%84%EF%B8%8F/)) do get their backlinks.

This is my setup in mkdocs.yml:
``` yml
# mkdocs.yml
site_name: Icewind Dale
site_url: https://lbeukennoot.github.io/dnd-icewind-dale/
repo_url: https://github.com/LBeukennoot/dnd-icewind-dale
repo_name: LBeukennoot/dnd-icewind-dale

nav:
  - Home: index.md
  - Sessions: 
    - Session 1: Sessions/Session 1
    - Session 2: Sessions/Session 2
    - Session 3: Sessions/Session 3
    - Session 4: Sessions/Session 4
    - Session 5: Sessions/Session 5
    - Session 6: Sessions/Session 6
    - Session 7: Sessions/Session 7
  - Villages:
    - Bremen ❄: Villages/Bremen ❄%EF%B8%8F/
    - Bryn Shander ❄: Villages/Bryn Shander ❄%EF%B8%8F/
    - Caer-Denival ❄: Villages/Caer-Denival ❄%EF%B8%8F/
    - Caer-Konig ❄: Villages/Caer-Konig ❄%EF%B8%8F/
    - Dougan's Hole: Villages/Dougan's Hole/
    - Easthaven ❄: Villages/Easthaven ❄%EF%B8%8F/
    - Good Mead: Villages/Good Mead/
    - Lonelywood ❄: Villages/Lonelywood ❄%EF%B8%8F/
    - Targos ❄: Villages/Targos ❄%EF%B8%8F/
    - Termalaine ❄: Villages/Termalaine ❄%EF%B8%8F/

theme:
    name: 'material'
    palette:

    # Light mode
    - media: "(prefers-color-scheme: light)"
      scheme: default
      primary: blue
      accent: indigo
      toggle:
        icon: material/toggle-switch-off-outline
        name: Switch to dark mode

    # Dark mode
    - media: "(prefers-color-scheme: dark)"
      scheme: slate
      primary: light blue
      accent: blue
      toggle:
        icon: material/toggle-switch
        name: Switch to light mode

    include_sidebar: true

    features:
      - navigation.path
      - navigation.sections

# Extensions
markdown_extensions:
  - footnotes
  - pymdownx.arithmatex:
      generic: true
  # - pymdownx.superfences
  # - pymdownx.details
  # - pymdownx.magiclink
  - pymdownx.tasklist:
      custom_checkbox: true
  - def_list
  - admonition
  - attr_list
  - md_in_html
  - pymdownx.details
  - pymdownx.superfences
  - pymdownx.critic
  - pymdownx.caret
  - pymdownx.keys
  - pymdownx.mark
  - pymdownx.tilde
  - pymdownx.emoji:
      emoji_index: !!python/name:material.extensions.emoji.twemoji
      emoji_generator: !!python/name:material.extensions.emoji.to_svg
  - pymdownx.superfences:
      custom_fences:
        - name: mermaid
          class: mermaid
          format: !!python/name:pymdownx.superfences.fence_code_format

  - toc:
      permalink: true

plugins:
  - backlinks_section:
      add_to_toc: false
      hide_if_empty: false
      title: "Links mentioned"
      description: ""
      description_no_links: "No links."
      ignore_links_to:
        - index.md
  - search
  - roamlinks
  - blog
  - rss:
      match_path: sessions/.* 
      date_from_meta:
        as_creation: date
      categories:
        - categories
        - tags

extra_javascript:
  - javascripts/mathjax.js
  - https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js
  - https://unpkg.com/mermaid/dist/mermaid.min.js 
```

My theory is that this has something to do with the amount of references. I hope this is enough info and am eager to hear about possible solutions!

**Comment by six-two** [comment]
@LBeukennoot Thank you for opening the issue and supplying a project, where I can reproduce the issue. I had a few suspicions, but so far all of them turned out wrong. I will keep investigating and update you if I find the cause or a workaround.

**Comment by six-two** [comment]
@LBeukennoot I think I may have found and fixed the cause. My path normalization code had a bug that normalized  `../Quests/` into `Quests/quests` instead of the expected all-lowercase `quests/quests`. The fix works in my local clone of your site. Can you confirm that it works for you too?

For testing you can replace the `pip install mkdocs-backlinks-section-plugin` in your `.github/workflows/ci.yml` with `pip install git+https://github.com/six-two/mkdocs-backlinks-section-plugin`.

**Comment by LBeukennoot** [comment]
Thank you for the quick fix! This solution fixes the problem and the backlinks are now visible on all pages. 👍🏻

**Comment by six-two** [comment]
@LBeukennoot Thank you for the feedback. I now released the fix in version 0.0.6, so you should be able to revert to the installation with `pip install mkdocs-backlinks-section-plugin`.
